cli/tests/e2e/snapshots/test_dependency_aware_rules/test_dependency_aware_rules/rulesdependency_awarelog4shell.yaml-dependency_awarelog4j/results.txt

```diff
@@ -39,19 +39,6 @@ SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERS
               "line": 33,
               "offset": 1035
             },
-            "propagated_value": {
```

Comment:
this seems unrelated. cc @mmcqd

Reply:
Is is related to this change but unrelated to SSC:

> Unfortunately, this will have some side-effects on sym-prop, because we kill sym-prop values at JOIN points, so any sym-prop values are lost after an if-statement. Sym-prop is experimental so that's OK.

Reply:
See also this test I added: https://github.com/returntocorp/semgrep/blob/38baaabc8d7d45ba2cf3d5c0e10ef2777c7a7826/tests/rules/sym_prop_no_merge4.java